src/once.js

```diff
@@ -0,0 +1,26 @@
+import {decorate} from './private/utils';
+
+function handleDescriptor(target, key, descriptor) {
+  const fn = descriptor.value;
+
+  if (typeof fn !== 'function') {
+    throw new SyntaxError(`@once can only be used on functions, not: ${fn}`);
+  }
+
+  let returnedValue;
+
+  return {
+    ...descriptor,
+    value() {
+      if (!returnedValue) {
```

Comment:
Should use a second boolean variable to check if the function has been run. If the first run of the function returns a falsy value, it allows the function to run multiple times. An example would be a function that has no return value would always run.